tests/test_ilastik/test_workflows/testCarvingGui.py

```diff
@@ -25,24 +25,15 @@
 logger.addHandler(logging.StreamHandler(sys.stdout))
 logger.setLevel(logging.DEBUG)
 
-import platform
 import pytest
 
 
 if os.environ.get("ON_CIRCLE_CI"):
     pytest.skip("These tests are too flaky on circleCI", allow_module_level=True)
+elif not is_mac_py_compatible():
```

Comment:
Hmm, now looking at this I wonder whether it'd be better to check for `os.environ.get("VOLUMINA_SHOW_3D_VIDGET", "1") == "0"`...